src/main/java/org/apache/datasketches/tuple/CompactSketch.java

```diff
@@ -109,13 +111,17 @@ private enum Flags { IS_BIG_ENDIAN, IS_READ_ONLY, IS_EMPTY, IS_COMPACT, IS_ORDER
           offset += classNameLength;
         }
         hashArr_ = new long[count];
+
         for (int i = 0; i < count; i++) {
           hashArr_[i] = mem.getLong(offset);
           offset += Long.BYTES;
         }
         for (int i = 0; i < count; i++) {
           offset += readSummary(mem, offset, i, count, deserializer);
         }
+      } else {
```

Comment:
This seems newly added. Did it fix a bug by handling a path that we previously ignored?

Reply:
This was caught by SpotBugs that needed to be final, which means it needs to be initialized whether the sketch has entries or not.  summaryArr_ is along for the ride, just to make sure it has a valid initialization too.